test/benchmarks.cpp

```diff
@@ -419,3 +386,37 @@ TEST_CASE("test_qft_superposition_round_trip")
         },
         true, true, testEngineType == QINTERFACE_QUNIT);
 }
+
+TEST_CASE("test_grover", "[grover]")
```

Comment:
Don't change the location in code of this, as it inhibits change analysis.

Reply:
Done.